backend/api/Program.cs

```diff
@@ -48,8 +53,41 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
                     config.AddEnvironmentVariables();
                     config.AddCommandLine(args);
                 })
+                .UseSerilog()
                 .UseUrls(config.GetValue<string>("ASPNETCORE_URLS"))
                 .UseStartup<Startup>();
         }
+
+        private static void ConfigureLogging()
```

Comment:
I find it odd we need to configure logging here.  I'm surprised we wouldn't configure it in the `Startup.ConfigureServices(...)` function.  As that is where we setup DI.